Remove Conda from script and set path for DB as defined in module file
Modify the script to remove the hard-coded number of CPUs and MEM according to your 
job-scheduler!
Author: J. Sassmannshausen (Imperial College London/UK)
idiff --git a/RoseTTAFold2NA-0.2.orig/input_prep/make_protein_msa.sh b/RoseTTAFold2NA-0.2/input_prep/make_protein_msa.sh
index 1f7a260..5185381 100755
--- a/RoseTTAFold2NA-0.2.orig/input_prep/make_protein_msa.sh
+++ b/RoseTTAFold2NA-0.2/input_prep/make_protein_msa.sh
@@ -10,8 +10,8 @@ CPU="$4"
 MEM="$5"
 
 # sequence databases
-DB_UR30="$PIPEDIR/UniRef30_2020_06/UniRef30_2020_06"
-DB_BFD="$PIPEDIR/bfd/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt"
+DB_UR30="${DB_DIR}/UniRef30_2020_06/UniRef30_2020_06"
+DB_BFD="${DB_DIR}/bfd/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt"
 
 # setup hhblits command
 HHBLITS_UR30="hhblits -o /dev/null -mact 0.35 -maxfilt 100000000 -neffmax 20 -cov 25 -cpu $CPU -nodiff -realign_max 100000000 -maxseq 1000000 -maxmem $MEM -n 4 -d $DB_UR30"
@@ -84,4 +84,4 @@ fi
 if [ ! -s ${out_prefix}.msa0.a3m ]
 then
     cp $prev_a3m ${out_prefix}.msa0.a3m
-fi
\ No newline at end of file
+fi
diff --git a/RoseTTAFold2NA-0.2.orig/input_prep/make_rna_msa.sh b/RoseTTAFold2NA-0.2/input_prep/make_rna_msa.sh
index d731ee7..03fbdec 100755
--- a/RoseTTAFold2NA-0.2.orig/input_prep/make_rna_msa.sh
+++ b/RoseTTAFold2NA-0.2/input_prep/make_rna_msa.sh
@@ -15,7 +15,7 @@ fi
 CPU="$4"
 MEM="$5"
 
-RNADBDIR="$PIPEDIR/RNA"
+RNADBDIR="${DB_DIR}/RNA"
 
 # databases
 db0="$RNADBDIR/Rfam.cm";
diff --git a/RoseTTAFold2NA-0.2.orig/run_RF2NA.sh b/RoseTTAFold2NA-0.2/run_RF2NA.sh
index add3c8d..1815bdd 100755
--- a/RoseTTAFold2NA-0.2.orig/run_RF2NA.sh
+++ b/RoseTTAFold2NA-0.2/run_RF2NA.sh
@@ -3,18 +3,9 @@
 # make the script stop when error (non-true exit code) occurs
 set -e
 
-############################################################
-# >>> conda initialize >>>
-# !! Contents within this block are managed by 'conda init' !!
-__conda_setup="$('conda' 'shell.bash' 'hook' 2> /dev/null)"
-eval "$__conda_setup"
-unset __conda_setup
-# <<< conda initialize <<<
-############################################################
-
 SCRIPT=`realpath -s $0`
 export PIPEDIR=`dirname $SCRIPT`
-HHDB="$PIPEDIR/pdb100_2021Mar03/pdb100_2021Mar03"
+HHDB="${DB_DIR}/pdb100_2021Mar03/pdb100_2021Mar03"
 
 CPU="8"  # number of CPUs to use
 MEM="64" # max memory (in GB)
@@ -22,8 +13,6 @@ MEM="64" # max memory (in GB)
 WDIR=`realpath -s $1`  # working folder
 mkdir -p $WDIR/log
 
-conda activate RF2NA
-
 # process protein (MSA + homology search)
 function proteinMSA {
     seqfile=$1

